### ISSUE TITLE
extension does not work in pre-38 master

### ISSUE THREAD

**Issue by mackenza** [peterflynn/reasonable-comments#6]
I don't know if it's appropriate to log issues on a running Brackets from master but I thought you might want to know that the extension doesn't seem to work as of Mar 28th's commits.  I am trying to debug but no luck so far.  Could be the enter event handler, but I am not that experienced with JS.

If this is too premature to log, please go ahead and close. ;)

**Comment by mackenza** [comment]
I think I figured it out...

``` javascript
    function handleEnterKey(editor) {
        var cursor = editor.getCursorPos();
        var token = editor._codeMirror.getTokenAt(cursor);
//        console.log(token);

        if (token.className === "comment") {
```

I uncommented the `console.log(token);` and noted that it returned `type: comment` not `className: comment` so I change the test to:

``` javascript
    function handleEnterKey(editor) {
        var cursor = editor.getCursorPos();
        var token = editor._codeMirror.getTokenAt(cursor);
//        console.log(token);

        if (token.type=== "comment") {
```

and now it seems to work.

I wasn't sure if a pull-request was needed since this seems to be an upgrade from 37 to 38-ish issue (CM4?).

**Comment by mackenza** [comment]
and... from the CodeMirror upgrading docs...

> The className property in the return value of getTokenAt, which has been superseded by the type property, is also no longer present.
> http://codemirror.net/doc/upgrade_v4.html

I think I will mention this on the GGroups list as it's not in the Brackets upgrade guide.

**Comment by peterflynn** [comment]
Fixed in version 1.3
